apps/game/lib/game/cup.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Cup do
+  @moduledoc """
+  generat a hand of n dice in a cup
+  """
+
+  @spec new(integer) :: [integer]
+  def new(n, acc \\ [])
+  def new(0, acc), do: acc
+  def new(n, acc) do
+    new(n-1, [:rand.uniform(6) | acc])
+  end
```

Comment:
Hm. I don't think that function capture will work though, will it... because no arguments are used? I really should learn Elixir better sometime. Maybe write yourself a separate function:

`def rolldie(), do: rand.uniform(6)`